utils.c

```diff
@@ -71,23 +72,25 @@ gboolean janus_strcmp_const_time(const void *str1, const void *str2) {
 }
 
 guint32 janus_random_uint32(void) {
-	return g_random_int();
+	guint32 ret = 0;
+	if (RAND_bytes((void *)&ret, sizeof(ret)) != 1) {
```

Comment:
Code style: there shouldn't be a space between `if` and the bracket.

Reply:
done